examples/form/src/main.rs

```diff
@@ -51,15 +54,25 @@ async fn show_form() -> Html<&'static str> {
             </body>
         </html>
         "#,
-    )
+        errors.unwrap_or_default()
+    ))
 }
 
-#[derive(Deserialize, Debug)]
+#[derive(Deserialize, Debug, Validate)]
 struct Input {
+    #[validate(length(min = 2))]
     name: String,
+    #[validate(email)]
     email: String,
 }
 
-async fn accept_form(Form(input): Form<Input>) {
+async fn accept_form(Form(input): Form<Input>) -> Html<String> {
     dbg!(&input);
+    match input.validate() {
+        Ok(_) => String::from("Form submitted successfully!").into(),
+        // Here the validation errors are presented as-is. In a more realistic
+        // application we would need to customize/process the validation errors to be able to
```

Comment:
'need' is a strong word 😄 

```suggestion
        // application we would customize / process the validation errors to be able to
```